cmd/tempo-vulture/main.go

```diff
@@ -112,7 +122,6 @@ func main() {
 			// query the trace
 			metrics, err := queryTempoAndAnalyze(tempoQueryURL, hexID)
 			if err != nil {
-				glog.Error("error querying Tempo ", err)
```

Comment:
This log can be removed because an error is already logged by `queryTempoAndAnalyze`.